### ISSUE TITLE
Need assistance with module continuous-integration

### ISSUE THREAD

**Issue by rosnermd** [dxc-technology/online-devops-dojo#75]
**Describe the issue**
Build fails after I fix the missing parenthesis.
 
**To Reproduce**
1. Step 6/8, after I fix the missing parenthesis a build kicks off
2. Jenkins starts to build
3. Error

![image](https://user-images.githubusercontent.com/8060458/99084151-98919f80-2594-11eb-9cb2-63449c62a697.png)

**Expected behavior**
Build is suppose to succeed
**Screenshots**

**Additional context**

**Comment by adamdmharvey** [comment]
Hi @rosnermd ! Excited you're taking the course.

Your error above indicates you have a foreign key mistake for ID # 7.  ID 7 in the types table should be the addition of a new type of animal, the horse 🐴 .  If this is failing, it may mean that your schema doesn't support horses yet.

> integrity constraint violation: foreign key no parent; FK_PETS_TYPES table: PETS value: 7

Looking at your code, these are the list of types you have defined:

https://github.com/rosnermd/pet-clinic/blob/b1203e82cbb9196b3332af11bc83f1d080318e11/src/main/resources/db/hsqldb/data.sql#L18-L24

There is no type 7 horse, so you wouldn't be able to add a named horse in the PR you refer above which is here:
https://github.com/rosnermd/pet-clinic/pull/3/files

Basically, your test failure is saying, "horse? what's a horse???"

Given that, I might suggest if you can check if you completed Step 3 which is to add the type horse to the schema first?  You shuold have a PR which completes that step, before moving on to adding named horses in Step 6.

Could you give that a try and see if it helps?

Best of luck!  🎉

**Comment by rosnermd** [comment]
Hi, 
Thank you for responding!  That's definitely the issue, however the instructions didn't say anything about modifying the schema.  Just to drop in the couple new insert lines (unless I missed it).

Can you provide the instructions on the schema update?
Thanks.

**Comment by adamdmharvey** [comment]
Sure!  So it's step 3 here in the version control module:

https://github.com/dxc-technology/online-devops-dojo/blob/master/online-devops-dojo/version-control/step3-nospell.md

Or just take the **Version Control** module directly from the main training page:

https://dxc-technology.github.io/about-devops-dojo/modules/tag/online-devops-dojo/

They essentially run in order.  There's a note in the **Continuous Improvement** module regarding having completed the Version Control module, but perhaps it's not explicit enough. Maybe that's the nuance here, did you go straight to CI without Version Control?  If so, not a problem, just try the Version Control module first, and if that helps, maybe it could be made more clear to help other students also.

**Comment by rosnermd** [comment]
I did skip straight to the CI Module....PBKAC.

Thanks!
